shotover-proxy/tests/cassandra_int_tests/functions.rs

```diff
@@ -1,40 +1,29 @@
 use crate::helpers::cassandra::{assert_query_result, run_query, CassandraConnection, ResultValue};
 
 async fn drop_function(session: &CassandraConnection) {
-    assert_query_result(
-        session,
-        "SELECT test_function_keyspace.my_function(x, y) FROM test_function_keyspace.test_function_table WHERE id=1;",
-        &[&[ResultValue::Int(4)]]
-    ).await;
+    assert_query_result(session, "SELECT test_function_keyspace.my_function(x, y) FROM test_function_keyspace.test_function_table WHERE id=1;", &[&[ResultValue::Int(4)]]).await;
```

Comment:
I know rustfmt isnt running on these files but can we keep manually formatting something like what rustfmt would do?
Its a lot easier to read with the function args split each on their own line instead of the entire function call on one line.